### PR TITLE
big catalog

### PR DESCRIPTION
**Pull request by leonid-levtseniuk**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,375 @@
...
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="header__block">
+        <ul class="header__links">
+          <li>
+            <a href="#1" class="is-active">Apple
```
Comment (aliceblunt8) [inline src/index.html:29]: let's make nav a separate block

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#1" class="nav__link is-active">Apple
```

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,375 @@
...
+            </a>
+          </div>
+          <div class="card__rating">
+            <div class="card__stars">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
```
Comment (aliceblunt8) [inline src/index.html:207]: modifier class? you should reuse stars block from the previous task

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]
please get stars task approved and use everything fixed from previous PRs here

**Comment by leonid-levtseniuk** [comment]
> please get stars task approved and use everything fixed from previous PRs here

Ok, sorry for that

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**